.eslintrc.cjs

```diff
@@ -9,7 +9,8 @@ module.exports = {
       rules: {
         "@typescript-eslint/unbound-method": 0,
         "@typescript-eslint/explicit-module-boundary-types": 0,
-        "@typescript-eslint/no-unused-vars": 2
+        "@typescript-eslint/no-unused-vars": 2,
+        "import/no-cycle": 2
```

Comment:
Any reason to not use [`@guardian/eslint-config-typescript`](https://github.com/guardian/csnx/tree/main/libs/%40guardian/eslint-config-typescript)? Appreciate it'll likely be a bigger change than desired for now.

I'd recommend adding `ignoreExternal` to the `import/no-cycle` config to improve execution time. See https://github.com/guardian/csnx/pull/60.

Reply:
IIUC we extend that config in the line above – wonder if we're using a version that predates us adding the rule? Be nice to update it in a separate PR.

Reply:
> IIUC we extend that config in the lines above

Ah! I'll admit to being lazy, and not checking prior to the suggestion!

Reply:
Ace. We can't yet add that rule in this PR, so I'll resolve this now in anticipation of #267.